cts/CIB.py

```diff
@@ -367,7 +367,7 @@ def add_resources(self):
         # promotable clone resource
         s = Resource(self.Factory, "stateful-1", "Stateful", "ocf", "pacemaker")
         s.add_op("monitor", "15s", timeout="60s")
-        s.add_op("monitor", "16s", timeout="60s", role="Master")
+        s.add_op("monitor", "16s", timeout="60s", role="Main")
```

Comment:
Unfortunately this particular usage point is a deeply embedded standard across multiple software projects, so we are unable to change it at this time. There is a plan to start a transition in the first half of next year, but for now this must remain as it is.